tests/drum/test_typeschema_validation_unit.py

```diff
@@ -0,0 +1,32 @@
+from datarobot_drum.drum.typeschema_validation import NumColumns, Conditions
```

Comment:
i think it's time to get rid of `test_units.py` file and start using a better directory structure.  please do `tests/unit/drum/test_typeschema_validation.py` and move _all_ the appropriate tests from `test_units.py` over to here.  also, since this is a specific test for NumColumns, please put it in `test_...py/TestNumColumns::test_init_raises_value_error_for_bad_column_numbers` or something to that effect